x-pack/test/security_solution_api_integration/test_suites/genai/evaluations/trial_license_complete_tier/evaluations.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  API_VERSIONS,
+  ELASTIC_AI_ASSISTANT_EVALUATE_URL,
+  PostEvaluateBody,
+} from '@kbn/elastic-assistant-common';
+import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
+import {
+  loadConnectorsFromEnvVar,
+  loadLangSmithKeyFromEnvVar,
+} from '../../../../scripts/genai/vault/manage_secrets';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+import {
+  clearKnowledgeBase,
+  deleteTinyElser,
+  installTinyElser,
+  setupKnowledgeBase,
+} from '../../knowledge_base/entries/utils/helpers';
+
+import { MachineLearningProvider } from '../../../../../functional/services/ml';
+import { routeWithNamespace } from '../../../../../common/utils/security_solution';
+import { loadEvalKnowledgeBaseEntries } from '../data/kb_entries';
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertest');
+  const log = getService('log');
+  const es = getService('es');
+  const ml = getService('ml') as ReturnType<typeof MachineLearningProvider>;
+  const esArchiver = getService('esArchiver');
+
+  /**
+   * Results will be written to LangSmith for project associated with the langSmithAPIKey, then later
+   * aggregated in the below tracking sheet:
+   * https://docs.google.com/spreadsheets/d/1kDNu98XR4eMIlKNq2iHlx5lVS5Npzm9cyDLyUVroiP8/edit?gid=0#gid=0
+   *
+   * Note: Suite is disabled in `.buildkite/ftr_security_stateful_configs.yml` as it has its own
+   * weekly pipeline located at `.buildkite/pipelines/security_solution/gen_ai_evals.yml`
+   */
+  describe('@ess Basic Security AI Assistant Evaluations', () => {
+    before(async () => {
+      await installTinyElser({ ml, es, log });
+      await setupKnowledgeBase(supertest, log);
+      await esArchiver.load(
+        'x-pack/test/functional/es_archives/security_solution/attack_discovery_alerts'
+      );
+    });
+
+    after(async () => {
+      await deleteTinyElser({ ml, es, log });
+      await esArchiver.unload(
+        'x-pack/test/functional/es_archives/security_solution/attack_discovery_alerts'
+      );
+    });
+
+    afterEach(async () => {
+      await clearKnowledgeBase(es);
+    });
+
+    describe('Run Evaluations', () => {
+      const buildNumber = process.env.BUILDKITE_BUILD_NUMBER;
+      const defaultEvalPayload: PostEvaluateBody = {
+        runName: `Eval Automation${buildNumber ? ' - ' + buildNumber : ''}`,
+        graphs: ['DefaultAssistantGraph'],
+        datasetName: 'Sample Dataset',
+        connectorIds: Object.keys(loadConnectorsFromEnvVar()),
+        evaluatorConnectorId: 'gpt-4o',
```

Comment:
It would be nice if this was configurable via some pipeline input param, but not 100% on how we should go about doing that just yet so ended up locking to `4o` for the time being.